src/plugins/on/test.js

```diff
@@ -152,12 +152,13 @@ describe(".on(event, fn)", function () {
         done();
       }
     });
-    u('.cb-wrapper').on('change', '.cb', function () {
+    base.find('.cb-wrapper').on('change', '.cb', function () {
```

Comment:
Same here, why is this change needed?

Reply:
I made this change to my test to better match the style of the existing tests. I noticed after the fact that none of the I other tests were directly using u().